dnsdist/dnsdist.conf.j2

```diff
@@ -39,11 +38,25 @@ addTLSLocal("[::]", ssl_cert, ssl_key, { numberOfStoredSessions=0, reusePort=tru
 addTLSLocal("[::]", ssl_cert, ssl_key, { numberOfStoredSessions=0, reusePort=true, tcpFastOpenQueueSize=20, minTLSVersion="tls1.1" })
 addTLSLocal("[::]", ssl_cert, ssl_key, { numberOfStoredSessions=0, reusePort=true, tcpFastOpenQueueSize=20, minTLSVersion="tls1.1" })
 
+-- listen for DoQ on external interface
+addDOQLocal("0.0.0.0", ssl_cert, ssl_key, { reusePort=true })
+addDOQLocal("0.0.0.0", ssl_cert, ssl_key, { reusePort=true })
+addDOQLocal("[::]", ssl_cert, ssl_key, { reusePort=true })
+addDOQLocal("[::]", ssl_cert, ssl_key, { reusePort=true})
+
 -- listen for DoH on localhost for reverse proxy
 addDOHLocal("127.0.0.1:445", nil, nil, "/dns-query", { numberOfStoredSessions=0, reusePort=true, trustForwardedForHeader=true })
 addDOHLocal("127.0.0.1:445", nil, nil, "/dns-query", { numberOfStoredSessions=0, reusePort=true, trustForwardedForHeader=true })
 addDOHLocal("127.0.0.1:445", nil, nil, "/dns-query", { numberOfStoredSessions=0, reusePort=true, trustForwardedForHeader=true })
 addDOHLocal("127.0.0.1:445", nil, nil, "/dns-query", { numberOfStoredSessions=0, reusePort=true, trustForwardedForHeader=true })
+addDOHLocal("127.0.0.1:445", nil, nil, "/dns-query", { numberOfStoredSessions=0, reusePort=true, trustForwardedForHeader=true })
```

Comment:
Are those needed? I would be in favor in removing those threads again in prod and therefore in this PR as well... 🙄